packages/kbn-grid-layout/grid/grid_layout.test.tsx

```diff
@@ -112,17 +115,18 @@ describe('GridLayout', () => {
       drop(panel1DragHandle);
       expect(getAllThePanelIds()).toEqual([
         'panel2',
+        'panel1',
         'panel5',
         'panel3',
         'panel7',
-        'panel1',
-        'panel8',
         'panel6',
+        'panel8',
         'panel4',
         'panel9',
         'panel10',
```

Comment:
Collision resolution is different now, so the resulting layout looks a bit different - the panels are still being rendered top-bottom, left-right though.